src/pybind/mgr/orchestrator.py

```diff
@@ -333,6 +333,17 @@ def __init__(self):
         # The type of service (osd, mon, mgr, etc.)
         self.service_type = None
 
+        # Service version that was deployed
+        self.version = None
+
+        # Location of the service configuration when stored in rados
+        # object. Format: "<pool_name>/<object_name>"
+        self.rados_config_location = None
```

Comment:
What about namespaces here? FWIW, ganesha has crafted its own rados:// URL type for use internally:

```rados://<pool>/[<namespace/>]<object>```

A 2 component URL there means that the namespace is the default one. Maybe this should be displayed in a similar format?

Reply:
I agree, will change the comment accordingly. Should we also keep the `rados://' part?

Reply:
I'd be :+1: for using a full url here